src/sentry/migrations/0566_remove_cron_missed_margins_zero.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.20 on 2023-09-20 00:47
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def remove_missed_margins_zero(apps, schema_editor):
+    Monitor = apps.get_model("sentry", "Monitor")
+    for monitor in RangeQuerySetWrapperWithProgressBar(Monitor.objects.filter()):
```

Comment:
nit: `.filter()` with no parameters i believe is equivalent to `.all()`, but i prefer `.all` since its more explicit